compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/MockDesugar.java

```diff
@@ -136,16 +149,96 @@ private BLangFunction getOriginalFunction(String functionName) {
         return null;
     }
 
+    private BInvokableSymbol getImportFunction(String functionName, String packageName) {
+        BInvokableSymbol importFunction = null;
```

Comment:
`importFunction` hides the field declared at L68, Shall we use another name for this?
Also, initializer `null` is redundant.

Reply:
Changed the name for the BInvokableSymbol